store/root/store.go

```diff
@@ -158,6 +165,11 @@ func (s *Store) GetLatestVersion() (uint64, error) {
 }
 
 func (s *Store) Query(storeKey string, version uint64, key []byte, prove bool) (store.QueryResult, error) {
+	if s.telemetry != nil {
+		now := time.Now()
+		s.telemetry.MeasureSince(now, "root_store", "query")
```

Comment:
Rabbit, I opted to avoid using a `NoOpMetrics` types because in such I case, I want to avoid `time.Now()` calls. Using `!= nil` check allows us to only call `time.Now()` when metrics are set. I.e. `!= nil` is essentially another way to do no-op.